src/main/java/com/couchbase/lite/replicator/RemoteRequest.java

```diff
@@ -296,21 +298,22 @@ protected void executeRequest(OkHttpClient httpClient, Request request) {
                 Log.w(TAG, "%s: executeRequest() Exception: %s.  url: %s", e, this, e, url);
                 error = e;
             }
-            respondWithResult(fullBody, error, response);
+            respondWithResult(contentBody, contentSize, error, response);
```

Comment:
As we also pass the response object around, is it possible to get the content size from the response object instead of passing contentSize?